src/openassets.es6

```diff
@@ -29,14 +29,16 @@ module.exports = function(robot) {
   // Address Book
   //
 
+  let robotKeyword = process.env.OA_BOT_KEYWORD || 'kredits';
```

Comment:
hmm. ideally I would not make it default and keep it generic. This is just to not break the current setup. 

and actually I think I should also move this on a room level. To be able to configure the assets per room. (for example have devkredits in #kosmos-dev and kredits in #kosmos) 

What do you think? should we move the config then in the hubot-brain?